bin/test/errors/json-errors.t

```diff
@@ -4,10 +4,14 @@
 
 Empty json file:
   $ mustache empty.json foo.mustache
-  Fatal error: exception Ezjsonm.Parse_error(870828711, "JSON.of_buffer expected JSON text (JSON value)")
-  [2]
+  mustache: internal error, uncaught exception:
+            Ezjsonm.Parse_error(870828711, "JSON.of_buffer expected JSON text (JSON value)")
```

Comment:
It's not required for this PR, but it would be nice to pretty print these exceptions.

Reply:
It's on my mind. I think that giving nice error messages for those would require getting the error location in the JSON file, which may require an upstream improvement to Ezjsonm or the use of Jsonm directly.